shared/test/otSdkDriver_test.js

```diff
@@ -223,6 +243,46 @@ describe("loop.OTSdkDriver", function() {
 
       sinon.assert.calledWithExactly(driver._noteSharingState, "browser", true);
     });
+
+    it("should call onScreenSharePublishComplete", function() {
```

Comment:
Lets wrap these in a describe, and structure them the same was as I described for the general publishing.